MTA-7237 - CLI installation (#385)
migtools/mta-documentation: docs/topics/mta-cli/proc_running-the-containerless-mta-cli.adoc

:_mod-docs-content-type: PROCEDURE

[id="running-the-containerless-mta-cli_{context}"]

> = Analyzing an application

Author: ```suggestion
= Analyzing an application in containerless mode
```

[role="_abstract"]

> By default, the {ProductFullName} command-line interface (CLI) runs in containerless mode. In this mode, the CLI runs assessments directly on your local machine by using the provided shell script, without requiring Podman or Docker.

Author: ```suggestion
By default, the {ProductFullName} command-line interface (CLI) runs in containerless mode. In this mode, the CLI uses the provided shell script to run assessments directly on your local machine, and it does not require Podman or Docker.
```

[NOTE]
====

> Containerless mode is the default for Java application analysis. To use container runtime instead, set the `--run-local` flag to `false`:

Author: ```suggestion
Containerless mode is the default for Java application analysis. To use a container runtime instead, set the `--run-local` parameter to `false`:
```

----
--run-local=false
----

> For applications written in languages other than Java, the analysis runs in container mode automatically.

Author: ```suggestion
For applications that are written in languages other than Java, the analysis automatically runs in container mode.
```
